docs/element-definitions.md

```diff
@@ -252,73 +281,102 @@ transitions.
 
 - `element`: category (string, fixed)
 - `meta`
-    - `classes` (array, fixed)
-        - (enum[string])
-            - api - Category is a API top-level group.
-            - resourceGroup - Category is a set of resource.
-            - dataStructures - Category is a set of data structures.
-            - scenario - Category is set of steps.
-            - transitions - Category is a group of transitions.
-            - authSchemes - Category is a group of authentication and authorization scheme definitions
+    - `classes` (Array Element)
+        - `content` (array, fixed-type)
+            - One Of
```

Comment:
This actually looks like a reasonable solution to the `OneOf` issue mentioned above as well.